components/mixins/profile/profile-privacy.schema.json

```diff
@@ -18,12 +18,13 @@
       "properties": {
         "xdm:optOutConsentLevel": {
           "type": "object",
-          "title": "Opt-out consent level",
-          "description": "Opt-out consent level.",
+          "title": "Opt-out consent level (deprecated)",
+          "description": "Opt-out consent level (deprecated).",
+          "meta:status": "deprecated",
           "properties": {
             "xdm:privacyOptOuts": {
-              "title": "Privacy opt-outs",
-              "description": "Encapsulates different opt-out types for data privacy.",
+              "title": "Privacy opt-outs (deprecated)",
+              "description": "Encapsulates different opt-out types for data privacy (deprecated).",
```

Comment:
Same deprecated comment here. Any field you want no longer used should have the "meta:status": "deprecated" attribute added.

Reply:
Also, is it correct that this mixin should stay around so a user can use the "identityPrivacyInfo" field? Or should this whole mixin be deprecated?

Reply:
This field is an element of another field, optOutConsentLevel, that I marked as deprecated. Do I need to mark all of its elements deprecated as well?

The RTCDP product is using the identityPrivacyInfo field. That is why I am not deprecating the whole file.